server.js

```diff
@@ -0,0 +1,221 @@
+const http = require('http');
+const fs = require('fs');
+const PORT = 8080;
+const querystring = require('querystring');
+let date = new Date().toUTCString();
+const admin = {
+    bri: 'password',
+}
+
+const server = http.createServer((req, res) => {
+    // console.log(req);
+    // console.log('reqmethod', req.method);
+    // console.log('requrl', req.url);
+    let body = '';
+    req.on('data', (chunk) => {
+        body += chunk;
+    });
+    req.on('error', (error) => {
+        console.error(error);
+    })
+    req.on('end', () => {
+        let urlType;
+        let URL = req.url;
+        // GET METHOD
+        if (req.method === 'GET') {
+            if (req.url === '/' || req.url === '') {
+                urlType = 'html';
+                URL = '/index.html';
+            } else {
+                urlType = req.url.split('.');
+                urlType = urlType[1];
+            }
+            fs.readdir(`./public/`, (err, files) => {
+                if (files.includes(URL.slice(1)) === false) {
+                    URL = `/404.html`;
+                    urlType = 'html';
+                }
+                fs.readFile(`./public${URL}`, (err, data) => {
+                    if (err) {
+                        return console.log('Could not write the file.');
+                    }
+                    res.writeHead(200, {
+                        'Date': `${date}`,
+                        'Content-Type': `text/${urlType}`,
+                        'Content-Length': data.toString().length,
+                    })
+                    res.write(data.toString());
+                    res.end();
+                })
+            })
+
+        }
+        // POST METHOD
+        if (req.method === 'POST') {
+            if (req.url === '/elements') {
+                let parsedInfo = querystring.parse(body);
+                const template = `<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="UTF-8">
+<title>The Elements - ${parsedInfo.elementName}</title>
+<link rel="stylesheet" href="/styles.css">
+</head>
+<body>
+    <h1>${parsedInfo.elementName}</h1>
+    <h2>${parsedInfo.elementSymbol}</h2>
+    <h3>${parsedInfo.elementAtomicNumber}</h3>
+    <p>${parsedInfo.elementDescription}</p>
+    <p><a href="/">back</a></p>
+</body>
+</html>`;
+                fs.writeFile(`./public/${parsedInfo.elementName}.html`, template, (err) => {
+                    if (err) {
+                        return console.log(err);
+                    }
+                    console.log('Filed Saved!');
```

Comment:
Don't console.log here, the client won't see it, instead you should move all the code below into this section so that everything below happens after you write the new element file